pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -16,22 +16,25 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "https://github.com/bazelbuild/bazel/releases/download/${version}/bazel-${version}-dist.zip";
-    sha256 = "0asmq3kxnl4326zhgh13mvcrc8jvmiswjj4ymrq0943q4vj7nwrb";
+    sha256 = "0csk3jzi3dvs24y9cmhlsbjyfv5pqddnqqkx3ywza58b1a4cbdbn";
   };
 
   sourceRoot = ".";
 
+  # bazel expects bash to find coreutils when starting from an empty environment
   postPatch = ''
+    makeWrapper "${bash}/bin/bash" "$out/bin/bash" --argv0 '$0' --prefix PATH : ${coreutils}/bin
```

Comment:
Might make sense to hide this `$out/bin/bash` somewhere other than `$out`, which might otherwise cause confusion when using `buildEnv` and friends (since a certain other package also contains a `bin/bash` 😄)